javascript/javascript1/week3/notetakingapp.js

```diff
@@ -0,0 +1,40 @@
+const notes = [];
+// this function adds an abject with keys and values to notes array
+
+function saveNote(content, id) {
+  if (typeof id === "number" && typeof content === "string") {
+    const noteApp = { content: content, id: id };
+    notes.push(noteApp);
+  }
+}
+//get a note by id
+function getNote(id) {
+  if (typeof id == "number") {
+    for (let i = 0; i < notes.length; i++) {
+      if (id === notes[i].id) {
+        return notes[i].content;
+      }
+    }
+    console.log("put the correct id number");
+    return;
+  }
+  console.log("Not a number");
+  return;
```

Comment:
you can log error one time after for loop. If you finished looping all notes and haven't found the note that has the id, that means, note not found
```js
function getNote(id) {
  if (typeof id !== "number") return;
    for (let i = 0; i < notes.length; i++) {
      if (id === notes[i].id) {
        return notes[i].content;
      }
    }
    console.error("note not found");
}
```